templates/static/shraga-silverstein.html

```diff
@@ -44,7 +44,7 @@ <h1>
                 <span class="int-he">הרב שרגא סילברסטיין סיים את לימודיו בקולג׳ ברוקלין בהצטיינות כבוגר הפקולטה ללימודי ספרות אנגלית וכן היה בוגר של ישיבת רבי חיים ברלין בברוקלין. הוא לימד במספר אוניברסיטאות בארצות הברית ובישראל וכן ניהל מספר ישיבות במהלך חייו.</span>
             </p>
             <p>
-                <span class="int-en">Since his untimely passing, Sefaria has been working with his family to obtain the rights to his unpublished works in order to make them widely available. Everything that is available on Sefaria is done so under the CC-BY license. We hope that this library will become a living memorial and make the works of Rabbi Silverstein as accesible as possible. What you see right now is just the beginning - we're going to keep adding his works to Sefaria and will keep this page updated as we do.</span>
+                <span class="int-en">Since his untimely passing, Sefaria has been working with his family to obtain the rights to his unpublished works in order to make them widely available. Everything that is available on Sefaria is done so under the CC-BY license. We hope that this library will become a living memorial and make the works of Rabbi Silverstein as accessible as possible. What you see right now is just the beginning - we're going to keep adding his works to Sefaria and will keep this page updated as we do.</span>
```

Comment:
This is effectively a copyright, but see notes about translations/typos. Someone probably needs to ask a lawyer for permission to fix this.